### ISSUE TITLE
`max` and negative numbers

### ISSUE THREAD

**Issue by fbs** [bpftrace/bpftrace#892]
Follow up from #834

The following program gives no output:

```
sudo bpftrace -e 'i:s:1 { @=max(-100); @=max(-5); exit() }'
Attaching 1 probe...

```

This is because CreateMapLookupElem returns 0 when the map is "empty". As -100 < 0 we don't have to do nothing and end up with a map we never write to.

One possible fix would be to have CreateMapLookupElem return INT_MIN if we're working with signed numbers and 0 when working with unsigned.

The printer will also have to be extended to properly print negative numbers.

**Comment by jordalgo** [comment]
This prints now but doesn't handle the negative number by default:
```
$ sudo bpftrace -e 'i:s:1 { @=max(-100); @=max(-5); exit() }'
Attaching 1 probe...


@: 18446744073709551611
```

**Comment by amscanne** [comment]
@jordalgo you fixed this in 1c9c3ce09f9de35cad3bdb362918f92619918b8f.

**Comment by jordalgo** [comment]
@amscanne Yup, seems to be working as expected.
